lib/elasticsearch.ts

```diff
@@ -70,14 +78,108 @@ function filter_old_indices(current_indices) {
 // was deleted or an error
 function delete_index(index) {
   return new Promise((resolve, reject) => {
-    request_es("del", `/${index}`).then(() => resolve(index)).catch(reject);
+    request_es_no_body("del", `/${index}`).then(() => resolve(index)).catch(reject);
   });
 }
 
 function delete_indices(indices) {
-  return Promise.all(_.map(indices, (index) => delete_index(index)));
+  return Promise.all(_.map(indices, delete_index));
 }
 
 export function clear_old_indices() {
   return get_indices().then(filter_old_indices).then(delete_indices);
 }
+
+function get_aliases() {
+  return new Promise((resolve, reject) => {
```

Comment:
do you need a new Promise here? I thought that returning the `then()` would chain this to the promise returned from `get_es` ?